src/components/modules/ApiManager.js

```diff
@@ -1,89 +1,94 @@
+  // GET all objects from tableName for the active user.
+  getAllSongs(tableName, userId) {
+    return fetch(
+      `${remoteURL}/${tableName}?userId=${userId}&_sort=lastUpdated&_order=desc`
+    ).then(result => result.json())
+  },
 
-    // GET all writers for a particular song.
-    getItemsForSpecificSong(tableName, songId) {
-      return fetch(`${remoteURL}/${tableName}?songId=${songId}`).then(result => result.json())
+  // GET all writers for a particular song.
+  getItemsForSpecificSong(tableName, songId) {
+    return fetch(`${remoteURL}/${tableName}?songId=${songId}`).then(result => result.json())
   },
 
-    // PATCH information to existing object.
-    patch(tableName, objectId, updatedObject) {
-      return fetch(`${remoteURL}/${tableName}/${objectId}`, {
-        method: "PATCH",
-        headers: {
-            "Content-Type": "application/json"
-        },
-        body: JSON.stringify(updatedObject)
-      }).then(results => results.json())
-    },
+  // PATCH information to existing object.
+  patch(tableName, objectId, updatedObject) {
+    return fetch(`${remoteURL}/${tableName}/${objectId}`, {
+      method: 'PATCH',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify(updatedObject),
+    }).then(results => results.json())
+  },
 
-    // PATCH information to existing object.
-    patchChords(songId, updatedObject) {
-      return fetch(`${remoteURL}/chords/?songId=${songId}`, {
-          method: "PATCH",
-          headers: {
-              "Content-Type": "application/json"
-          },
-          body: JSON.stringify(updatedObject)
-        }).then(results => results.json())
+  // PATCH information to existing object.
+  patchChords(songId, updatedObject) {
+    return fetch(`${remoteURL}/chords/?songId=${songId}`, {
+      method: 'PATCH',
+      headers: {
+        'Content-Type': 'application/json',
       },
-    
-    // DELETE an object from tableName.
-    delete(tableName, id) {
-      return fetch(`http://localhost:5002/${tableName}/${id}`, {
-          method: "DELETE"
-      })
-      .then(result => result.json())
-    },
+      body: JSON.stringify(updatedObject),
+    }).then(results => results.json())
+  },
+
+  // DELETE an object from tableName.
+  delete(tableName, id) {
+    return fetch(`http://localhost:5002/${tableName}/${id}`, {
+      method: 'DELETE',
+    }).then(result => result.json())
+  },
 
-    // DELETE corresponding objects for a song from tableName.
-    deleteChain(tableName, songId) {
-      return fetch(`http://localhost:5002/${tableName}?songId=${songId}`, {
-              method: "DELETE"
-      })
-      .then(result => result.json())
-    },
+  // DELETE corresponding objects for a song from tableName.
+  deleteChain(tableName, songId) {
+    return fetch(`http://localhost:5002/${tableName}?songId=${songId}`, {
+      method: 'DELETE',
+    }).then(result => result.json())
+  },
 
-    // POST a new object to tableName.  
```

Comment:
Oooo... I actually might have fucked this up. Something wasn't working for me, and I should not have committed this change. Going to revert it.